_layouts/home.html

```diff
@@ -3,7 +3,7 @@
  Copyright (c) 2016 StrongLoop, IBM, and Express Contributors
  License: MIT
 -->
-<html lang="{{ page.lang }}">
+<html class="light-mode" lang="{{ page.lang }}">
```

Comment:
Add light-mode by default

Reply:
Why not do it globally? I don't see the need to add it to just one class.

Reply:
The css is triggered by light-mode or dark-mode now, so it must have a theme class on the html. This just the default.

Reply:
```suggestion
<html lang="{{ page.lang }}">
```

Not adding default classes, but well, I see it's used elsewhere, so we would need to debug further. I'll be back soon, I'll keep removing prims from the page.

Reply:
We need someway to trigger the themes. It's pretty well based around this setup now, and is what allows the vars to work mostly in one file - style.css. Don't know it can be unwound - unless you see a way. 
Anyway, it's just a single class so the overhead is very low. And the JS used is very simple, add remove class stuff

Reply:
Yep, I can live with that.

Reply:
I'm testing it now that you say this to make sure. It might be added by the JS already so this class is _not_ needed. If this is the case then yes I'll remove it. Just need to test to make sure.

Reply:
Removed